crates/solidity/testing/sanctuary/src/reporting.rs

```diff
@@ -67,9 +66,8 @@ impl Reporter {
 
         self.parent.set_draw_target(ProgressDrawTarget::hidden());
 
-        std::io::stdout()
-            .write_all(&buffer.contents_formatted()[..])
-            .unwrap();
+        let contents = String::from_utf8(buffer.contents_formatted()).unwrap();
```

Comment:
Do we want to assert that the output is UTF-8? What's the reason behind that?

Reply:
It is expected to be valid UTF8. I split this to make it a bit clear.